packages/types/src/types/VSphereHostInventory.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * Represents the inventory of a vSphere host.
+ */
+export interface VSphereHostInventory {
```

Comment:
Can we drop `Inventory` suffix ? i.e. `VSphereDataStores` above don't use it and seems readable